shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -327,10 +328,16 @@ private Drawable getSplashScreenFromManifest() {
       Integer splashScreenId =
           metaData != null ? metaData.getInt(SPLASH_SCREEN_META_DATA_KEY) : null;
       return splashScreenId != null
-          ? Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP
+          ? Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
               ? getResources().getDrawable(splashScreenId, getTheme())
               : getResources().getDrawable(splashScreenId)
           : null;
+    } catch (Resources.NotFoundException e) {
+      Log.w(
+          TAG,
+          "Splash screen not found. Ensure the drawable exists and that it's valid."
+              + " This can also occur if the xml has unresolved theme references.");
```

Comment:
```suggestion
      Log.e(
          TAG,
          "Splash screen not found. Ensure the drawable exists and that it's valid."
              + " This can also occur if the xml has unresolved theme references.", e);
```